cloud/google/clients/computeservice.go

```diff
@@ -27,11 +26,7 @@ func NewComputeServiceForURL(client *http.Client, baseURL string) (*ComputeServi
 	if err != nil {
 		return nil, err
 	}
-	url, err := url.Parse(computeService.service.BasePath)
```

Comment:
Note for the reviewer: this was removed because after furthur research the BasePath is always to be simply an API "endpoint" and therefore cannot have any relative path that needs to be extracted from the URL.